pkgs/games/devilutionx/default.nix

```diff
@@ -107,6 +107,7 @@ stdenv.mkDerivation rec {
     mv devilutionx.app $out/Applications
   '' else ''
     install -Dm755 -t $out/bin devilutionx
+    install -Dm755 -t $out/bin devilutionx.mpq
```

Comment:
Is this needed for Darwin as well?

Reply:
I'm unable to test, but I don't think so. The macos dmg file from their release page includes files like `24-00.clx` manually without including the devilutionx.mpq file. Also, their [CMake/Assets.cmake](https://github.com/diasurgical/devilutionX/blob/7ea712428ebf3df5da58e107cc40e8db3825bebc/CMake/Assets.cmake#L162) demonstrates that Apple is an exception to building the MPQ.